test/staking/delegation.test.ts

```diff
@@ -593,68 +592,6 @@ describe('Staking::Delegation', () => {
       expect(await staking.getIndexerCapacity(indexer.address)).eq(0)
     })
 
-    it('should send delegation cut of query fees to delegation pool', async function () {
```

Comment:
Are we testing the delegation cut in a different place?

Reply:
yeah for some reason that was on a different file, I put everything together because it made more sense and reduced duplicating a bunch of code in both files.